src/translations/en.json

```diff
@@ -338,5 +338,10 @@
   "claim.calculateAge.util.days": "Day(s)",
   "claim.calculateAge.util.weeks": "Week(s)",
   "claim.calculateAge.util.months": "Month(s)",
-  "claim.calculateAge.util.years": "Year(s)"
+  "claim.calculateAge.util.years": "Year(s)",
+  "claim.pre-authorization": "Pre-Authorization Required",
+  "claim.pre-authorization.null" : "Any",
+  "claim.pre-authorization.true" : "Yes",
+  "claim.pre-authorization.false" : "No",
```

Comment:
To be consistent with other keys, use camel case: `preAuthorization` instead of `pre-authorization`